pkg/kinflate/util/manifestloader.go

```diff
@@ -33,6 +33,10 @@ type ManifestLoader struct {
 	FS fs.FileSystem
 }
 
+//This is a placeholder for the ManifestErrors type
+//TODO: Think about this struct and make is better
+type ManifestErrors []string
```

Comment:
shouldn't this be a list of errors rather than strings?

Reply:
`ManifestErrors` is a struct that we are going to define and use as the data store for all the errors we can get when loading a manifest.  It should be easy to add, group and print the erros. A list of errors could serve the basic need, but we want to think about a better struct.